packages/starrating/src/css/index.js

```diff
@@ -0,0 +1,79 @@
+import core from '@pluralsight/ps-design-system-core'
+
+export const BASE_CLASSNAME = '.psds-starrating'
+
+const hideVisually = {
+  border: 0,
+  clip: 'rect(1px, 1px, 1px, 1px)',
+  clipPath: 'polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px)',
+  height: 1,
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  whiteSpace: 'nowrap',
+  width: 1,
+
+  '&:active, &:focus': {
+    clip: 'auto',
+    clipPath: 'none',
+    height: 'auto',
+    overflow: 'visible',
+    position: 'static',
+    width: 'auto'
+  }
+}
+
+const resetButton = {
+  background: 'transparent',
+  border: 'none',
+  color: 'inherit',
+  font: 'inherit',
+  lineHeight: 'normal',
+  margin: 0,
+  overflow: 'visible',
+  padding: 0,
+  width: 'auto',
+  WebkitAppearance: 'none',
+
+  '&::-moz-focus-inner': {
+    border: 0,
+    padding: 0
+  }
+}
+
+export default {
+  [BASE_CLASSNAME]: {},
+
+  [`${BASE_CLASSNAME}__star`]: {
+    ...resetButton,
+    lineHeight: 0
+  },
+  [`${BASE_CLASSNAME}__star--active`]: {
+    color: core.colors.yellow
+  },
+  [`${BASE_CLASSNAME}__star--interactive`]: {
+    cursor: 'pointer'
+  },
+  [`${BASE_CLASSNAME}__star--theme-dark`]: {
```

Comment:
Thinking/writing about interpolation above makes me realize that we've used the vars in places like these elsewhere.  Do you feel like that has value?

Example: https://github.com/pluralsight/design-system/blob/master/packages/button/src/css/index.js#L43

My thought in the past has been that it ties our selectors to the code that drives those features, thus removing a magic string that can get out of sync if naming changes. 🤷‍♂️

Reply:
yeah i didn't think about that. i like using the the vars for this. i'll updated